<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot Arm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>115</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="modeSelectLayout">
     <item>
      <widget class="QLabel" name="modeSelectLabel">
       <property name="text">
        <string>Mode Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="modeSelectReal">
       <property name="text">
        <string>Simulation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="modeSelectSim">
       <property name="text">
        <string>Real</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="armView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>11</y>
      <width>661</width>
      <height>411</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="renderHints">
     <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="comPortTextField">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="comPortLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>COM Port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectionButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>479</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusText">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DISCONNECTED</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>126</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Control Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Drag Joint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Set Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="holdAngleField">
       <property name="text">
        <string>Hold Angle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
